tests/Doctrine/Tests/ORM/Functional/Ticket/GH10808Test.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\JoinColumn;
+use Doctrine\ORM\Mapping\OneToOne;
+use Doctrine\ORM\Mapping\Table;
+use Doctrine\ORM\UnitOfWork;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+use function get_class;
+
+/** @group GH10808 */
+class GH10808Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->createSchemaForModels(GH10808Appointment::class);
+        $this->createSchemaForModels(GH10808AppointmentChild::class);
```

Comment:
`createSchemaForModels`'s argument is variadic

```suggestion
        $this->createSchemaForModels(GH10808Appointment::class, GH10808AppointmentChild::class);
```

Reply:
Done